api/app/signals/apps/questionnaires/models/question_graph.py

```diff
@@ -46,3 +49,32 @@ def get_trigger_order(self, question):
 
     def set_trigger_order(self, question, ids):
         return self._set_model_order(question, Trigger, ids)
+
+    @property
+    def reachable_questions(self):
+        from signals.apps.questionnaires.models.utils import retrieve_reachable_questions
+        return retrieve_reachable_questions(self)
+
+    def draw(self, location='/'):
```

Comment:
See comment elsewhere about the default location and:

I think that we should be drawing to memory buffer and not to disk. Matplotlib can write to a `io.BytesIO` buffer. This would be similar to how we deal with resizing and re-encoding images and map tiles.

Reply:
When drawing to an `io.BytesIO` buffer I got some really strange behaviour that I was unable to fix. And because I will be on a long weekend I decided to make this PR mergeable by using a temporary directory instead.